changedetectionio/update_worker.py

```diff
@@ -74,6 +74,7 @@ def send_content_changed_notification(self, t, watch_uuid):
             n_object.update({
                 'watch_url': watch['url'],
                 'uuid': watch_uuid,
+                'screenshot': watch.get_screenshot() if watch.get('notification_screenshot') else False,
```

Comment:
```suggestion
                'screenshot': watch.get_screenshot(),
```

Returns False anyway

Reply:
This makes the checkbox useless

Reply:
@m42e but `watch.get_screenshot(),` returns False anyway, you dont need 'else False' or?

Reply:
It will return `False` if there is no screenshot available. But you want to be able to deactivate the screenshot for the notification.

Reply:
isnt this the notification object tho? nothing todo with the form?

Reply:
`notification_screenshot` is the storage of the Checkbox. `watch` is the watch object, offering a function `get_screenshot()` which will either return the file path of the screenshot or `False` if not available.
So the result would be:
- **True**: If the checkbox has been activated AND a screenshot is available
-  **False**: else (Either no screenshot or the checkbox has been deactivated)

Reply:
No, this is clearly just updating the notification object that goes into the queue to be processed

there is no need to run `get_screenshot` twice.

and then you are calling it again at

https://github.com/m42e/changedetection.io/blob/26efbb5ea528d3ecd5cf111041c298ce6efd2226/changedetectionio/notification.py#L106

so that's _4_ times you call `get_screenshot` all because you need to know if it's None

Please just make `get_screenshot` return `None` instead of `False`

and then remove this craziness where you call `get_screenshot` many times to check if its false/none etc